extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -240,6 +236,13 @@ export class AmpA4A extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(unusedOnLayout) {
+    // TODO(tdrl): Temporary, while we're verifying whether SafeFrame is an
+    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src from
+    // cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
+    this.preconnect.url(SAFEFRAME_IMPL_PATH);
+    //if (!this.config) {
```

Comment:
If you're commenting this out, I'm not sure you need the whole rearrangement of this function.